tools/src/bin/rust-indexer.rs

```diff
@@ -349,6 +349,40 @@ fn analyze_file(
     }
 }
 
+/// Uses heuristics to determine if the path is a Windows path,
+/// and if so, returns a Linux "equivalent". Not really equivalent,
+/// because there is no such thing, but equivalent enough for our
+/// purposes - that is, is_relative() returns a correct thing, and
+/// strip_prefix will work on relative paths.
+/// Note that this is a "linuxzed" path and not a "unixed" path
+/// this code should ever only really run inside the linux vagrant
+/// machine.
```

Comment:
Please link to the rls issue so that we can notice if / when it gets fixed :)

Reply:
Done